### PR TITLE
Als 2548 508 compliance for links opening in new tab

### PR DESCRIPTION
**Pull request by Gcolon021**
ALS-2548 - https://hms-dbmi.atlassian.net/browse/ALS-2548
Modal Screenshot

### CODE REVIEW

File: biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/tool-suite-help-view.hbs [hunk 1]
```diff
@@ -7,5 +7,6 @@
     <h5>Variable Distributions Tool</h5>
     <p>Visualize the distributions of phenotypic variables that have been added to the query for your filtered cohort. Categorical variable distributions are shown in pie charts and continuous variable distributions are shown as histograms.</p>
     <p>For more information about the tools, please refer to the
-        <a href="https://tinyurl.com/PIC-SURE-User-Guide" target="_blank" title='Clicking here will take you to the given link in another tab'>BioData Catalyst Powered by PIC-SURE User Guide</a>.</p>
+        <a href="https://tinyurl.com/PIC-SURE-User-Guide" target="_blank" aria-label="Clicking here will take you to the given link in another tab."
```
Comment (JamesPeck) [inline biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/tool-suite-help-view.hbs:10]:
```suggestion
        <a href="https://tinyurl.com/BDC-PIC-SURE-User-Guide" target="_blank" aria-label="Clicking here will take you to the given link in another tab."
```

Comment (JamesPeck) [reply biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/tool-suite-help-view.hbs:10]: I must have missed this link before. We updated to gitbook.

File: biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs [hunk 2]
```diff
@@ -12,7 +12,8 @@
                     will return aggregate counts per study and consent. From PIC-SURE Open Access you have the ability
                     to request access to studies you are currently not authorized to access. Visit the
                     <a href="https://tinyurl.com/PIC-SURE-User-Guide"
```
Comment (JamesPeck) [inline biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs:14]:
```suggestion
                    <a href="https://tinyurl.com/BDC-PIC-SURE-User-Guide"
```

**Review by JamesPeck — CHANGES_REQUESTED** [review]

File: biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js [hunk 3]
```diff
@@ -30,6 +27,24 @@ define(["handlebars", "text!overrides/footer.hbs"], function(HBS, template){
...
+						modal.displayModal(
+							linkView
+							, "Leaving BDC-PIC-SURE"
+							, () => {
+							}
+							, {handleTabs: true, width: "450px"}
+						);
+					});
+				});
+			}
```
Comment (JamesPeck) [inline biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js:47]: You could use https://github.com/hms-dbmi/biodatacatalyst-pic-sure/blob/master/biodatacatalyst-ui/src/main/webapp/picsureui/common/pic-sure-dialog-view.js istead of using a new view.

Comment (JamesPeck) [reply biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js:47]: Example: https://github.com/hms-dbmi/biodatacatalyst-pic-sure/blob/6cc894ddd26ed2fafd22ba3333a36383e5307b2d/biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/search-view.js#L79

File: biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs (hunk 2 above)
Comment (Gcolon021) [reply biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs:14]: I don't think this template is used anywhere. The router module includes it as a dependency, but doesn't use it anywhere.

File: biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js (hunk 3 above)
Comment (Gcolon021) [reply biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js]: I refactored the code and I now use the pic-sure-dialog-view. This allowed me to remove both of the link-redirect-warning files and will make the code more robust. Thanks for calling it out.

File: biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs (hunk 2 above)
Comment (JamesPeck) [reply biodatacatalyst-ui/src/main/webapp/picsureui/openPicsure/searchHelpTooltipOpen.hbs:14]: Lets delete it then

File: biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js [hunk 4]
```diff
@@ -30,6 +27,53 @@ define(["handlebars", "text!overrides/footer.hbs"], function(HBS, template){
...
 			this.$el.html(HBS.compile(template)());
+
+			if (session.isValid()) {
+				$(document).ready(function () {
+					$('a[target="_blank"]').on('click', function (event) {
+						event.preventDefault();
+
+						let closeModal = () => {
+							$("#modalDialog").hide();
+							$(".modal-backdrop").hide();
```
[the remark this answers is not in the document]
Comment (JamesPeck) [reply biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js:38]: Lets try and keep all the close logic in the modal.js file that way if it changes we don't need to change it in a bunch of places.

**Review by JamesPeck — CHANGES_REQUESTED** [review]

**Pull request merged**